CHANGELOG.rst

```diff
@@ -14,6 +14,10 @@ Change Log
 Unreleased
 ~~~~~~~~~~
 
+[4.1.0]- 2021-07-19
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+* Add Django 3.2 Support
```

Comment:
According to Jeremy, the versions `3.0` and `3.1` should also be mentioned in the changelog if we are adding them in the `Travis` now.